Doc/library/stdtypes.rst

```diff
@@ -1376,6 +1376,108 @@ objects that compare equal might have different :attr:`~range.start`,
    single: str (built-in class); (see also string)
    object: string
 
+.. _text-methods-summary:
+
+Text and Binary Sequence Type Methods Summary
+=============================================
+The following table summarizes the text and binary sequence types methods by
+category.
```

Comment:
Thanks, Martin.  You're right.  I wasn't quite sure where to place this overall and thought there would probably be suggestions, so I kept it vague.  There wasn't a section that combined both text and binary sequence types before and I didn't know if people would like having str and bytes (and bytearray) types all in one table or if there should be a separate table for each one under the existing sections.